packages/react-inbox/src/components/Messages/index.tsx

```diff
@@ -110,7 +110,7 @@ const Messages: React.ForwardRefExoticComponent<
                   <Message key={message.messageId} {...message} />
                 )
               )}
-              {isLoading && <Loading />}
+              {isLoading && messages?.length === 0 && <Loading />}
```

Comment:
why do we need to add this?

Reply:
just to make the inbox nicer.  that if you open, close and reopen it doesn't show the loading screen again...

Reply:
oh nice we had a couple of folks asking about that fluctuation